css/tailwind.css

```diff
@@ -0,0 +1,228 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  body {
+    font-family: "Roboto", sans-serif;
+  }
+
+  .navbar {
+    @apply flex flex-row items-center justify-between px-32 py-[8px];
+  }
+  .navbar__links {
+    @apply space-x-9;
+  }
+  .navbar__link {
+    @apply font-normal text-base text-[#1D2130];
+  }
+
+  .btn {
+    @apply bg-[#1D2130] rounded-xl py-[16px] px-[40px] text-white font-bold text-base text-center;
```

Comment:
declare this class `bg-[#1D2130]` use apply to reuse it =, to improve you consistance